CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # did:key driver ChangeLog
 
+## 0.6.0 - 2020-03-09
+
+### Changed
+- **BREAKING**: Use `x25519-key-pair` v2.0.0, changed fingerprint format
+  for X25519 keys.
+- Use `crypto-ld` v0.3.7.
```

Comment:
Was this supposed to be 3.7.0?

Reply:
@davidlehn whoops, yes :) thank you for the catch.

Reply:
(The actual `package.json` has the correct version.)

Reply:
Yup, no big deal.  Just needs a quick CHANGELOG update.